src/app/components/home/Hero.tsx

```diff
@@ -85,7 +87,10 @@ const Hero = () => {
           </div>
 
           <div className=" w-[50%] md:w-[20%] h-full flex items-center justify-end">
-            <Link href="https://app.decenterai.com/signup" className="hidden md:block">
+
+            <Link href="https://app.decenterai.com/explore" className="hidden md:block">
```

Comment:
don't do a refactor to https://app.decenterai.com. no explore. We cannot do this all the time,

Reply:
Do a refactor to `https://app.decenterai.com/.` not /explore

Reply:
@techyNonso